packages/developer-portal/src/components/settings/members/member-update-controls.tsx

```diff
@@ -203,41 +203,47 @@ export const MemberUpdateControls: FC<MemberUpdateControlsProps> = ({ member, re
-                </BodyText>
-              </Col>
-              <Col>
-                <Subtitle hasNoMargin>Status</Subtitle>
-                <BodyText hasGreyText hasNoMargin>
-                  {status === 'inProgress' ? 'Currently Configuring' : status === 'active' ? 'Active' : 'Disabled'}
-                </BodyText>
-              </Col>
-              <Col>
-                <Subtitle hasNoMargin>Created</Subtitle>
-                <BodyText hasGreyText hasNoMargin>
-                  {modified ? dayjs(created).format('DD/MM/YYYY HH:mm') : '-'}
-                </BodyText>
-              </Col>
-              <Col>
-                <Subtitle hasNoMargin>Last Updated</Subtitle>
-                <BodyText hasGreyText hasNoMargin>
-                  {modified ? dayjs(modified).format('DD/MM/YYYY HH:mm') : '-'}
-                </BodyText>
-              </Col>
-              <Col>
-                <DeleteAuthenticator
-                  authenticatorId={id}
-                  userId={userId}
-                  refreshAuthenticators={refreshAuthenticators}
-                />
-              </Col>
-            </Grid>
-          ))}
+          {authenticators && authenticators.length >= 1 ? (
+            authenticators?.map(({ type, modified, created, id, userId, status }) => (
+              <Grid className={cx(elMb11, elFadeIn, elMt11)} key={id}>
+                <Col>
+                  <Subtitle hasNoMargin>Authenticator Type</Subtitle>
+                  <BodyText hasGreyText hasNoMargin>
+                    {type === 'sms' ? 'Mobile SMS' : 'Authenticator App'}
+                  </BodyText>
+                </Col>
+                <Col>
+                  <Subtitle hasNoMargin>Status</Subtitle>
+                  <BodyText hasGreyText hasNoMargin>
+                    {status === 'inProgress' ? 'Currently Configuring' : status === 'active' ? 'Active' : 'Disabled'}
+                  </BodyText>
+                </Col>
+                <Col>
+                  <Subtitle hasNoMargin>Created</Subtitle>
+                  <BodyText hasGreyText hasNoMargin>
+                    {modified ? dayjs(created).format('DD/MM/YYYY HH:mm') : '-'}
+                  </BodyText>
+                </Col>
+                <Col>
+                  <Subtitle hasNoMargin>Last Updated</Subtitle>
+                  <BodyText hasGreyText hasNoMargin>
+                    {modified ? dayjs(modified).format('DD/MM/YYYY HH:mm') : '-'}
+                  </BodyText>
+                </Col>
+                <Col>
+                  <DeleteAuthenticator
+                    authenticatorId={id}
+                    userId={userId}
+                    refreshAuthenticators={refreshAuthenticators}
+                  />
+                </Col>
+              </Grid>
+            ))
+          ) : (
+            <PersistantNotification intent="primary" isInline isExpanded className={cx(elMt11, elMb11)}>
+              No authenticators configured for this user.
```

Comment:
this will be shown while `authenticatorsLoading` is true, should we use a loading indicator here instead?